bundle-ml/src/main/scala/ml/combust/bundle/BundleFile.scala

```diff
@@ -42,8 +42,17 @@ object BundleFile {
       case "file" =>
         (FileSystems.getDefault, FileSystems.getDefault.getPath(uriSafe.getPath))
       case "jar" =>
-        val zfs = FileSystems.newFileSystem(uriSafe, env)
-        (zfs, zfs.getPath("/"))
+        // handle resource in JAR path
```

Comment:
Can you add a test for this to make sure we aren't breaking anything?

Reply:
Sure. Added.